main/src/ca/uwaterloo/flix/language/ast/Name.scala

```diff
@@ -157,11 +157,12 @@ object Name {
   /**
     * Qualified Name.
     *
-    * @param namespace the namespace
-    * @param ident     the identifier.
-    * @param loc       the source location of the qualified name.
+    * @param namespace    the namespace
+    * @param ident        the identifier.
+    * @param loc          the source location of the qualified name.
+    * @param trailingDot  `true` if the qualified name ends with a dot.
```

Comment:
-> `true` if the qualified name ends with a dot (which implies this name is incomplete).